module/__init__.py

```diff
@@ -13,15 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import division, absolute_import
+from __future__ import absolute_import, division
 
 import ctypes.util
 import functools
 import platform
-import six
 import struct
 import weakref
 
+import six
```

Comment:
I think since you wrote this I dropped six, you can do the same in this patch.

Reply:
I might be better off rebasing then as there are lines that use `six` in my local copy. This is only showing in my commit as I think `isort` moved it.